conf.mk

```diff
@@ -1,6 +1,6 @@
 INSTALL_TARGETS += install-conf
 
-default_confs := 10-autoexec 40-escape 40-home_end 40-setprompt 40-sigint 90-hist 99-clear
+default_confs := 10-autoexec 20-completion 20-look_and_feel 40-escape 40-home_end 40-setprompt 40-sigint 90-hist 99-clear
```

Comment:
I don't think the completion-thing should be installed by default. It's highly dependent on personal taste and I can't even seem to find a setting I personally like…